sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -305,26 +285,6 @@ public void testReadWithQuery() throws Exception {
     pipeline.run();
   }
 
-  @Test
-  public void testReadWithValueProvider() throws Exception {
```

Comment:
This one was removed because the functionality is already covered by delegation from the different methods e.g. `withTable(String) -> withTable(ValueProvider)`